AUTHORS

```diff
@@ -0,0 +1,29 @@
+# This is the list of [Project Name] authors for copyright purposes. This does
```

Comment:
`[Project Name]` should be "spdx/tools-python" or similar.

Also it would be great if you could add the script used to create this into this PR, so it can be re-run as part of the release process.